README.md

```diff
@@ -40,6 +40,7 @@ _This image is made to be used in conjunction with a pre-configured SonarQube se
             -v "$(pwd):/usr/src" \
             lequal/sonar-scanner
     ```
+    This docker command is equivalent to `sonar-scanner -Dsonar.host.url="url of your SonarQube instance"`.
```

Comment:
Codacy found an issue: [[list-item-content-indent] Don’t use mixed indentation for children, remove 1 space](https://app.codacy.com/gh/cnescatlab/sonar-scanner/pullRequest?prid=6215690)